optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingBeyondThisTutorial/CloudBalancingBeyondThisTutorial-section.adoc

```diff
@@ -2,8 +2,8 @@
 = Beyond this Tutorial
 :imagesdir: ../..
 
-Now that this simple example works, try going further.
-Enrich the domain model and add extra constraints such as these:
+Now that this simple example works, you can try going further.
+For example, you can enrich the domain model and add extra constraints such as these:
 
-* Each `Process` belongs to a ``Service``. A computer might crash, so processes running the same service should be assigned to different computers.
-* Each `Computer` is located in a ``Building``. A building might burn down, so processes of the same services should be assigned to computers in different buildings.
+* Each `Process` belongs to a ``Service``. A computer might crash, so processes running the same service should (or must) be assigned to different computers.
```

Comment:
lets change "should" to "must".

The convention is to use "should" for soft constraints and "must" for hard constraints. That would definitely be a hard constraint.

Reply:
I didn't know it was a hard constraint in the code. Certainly changing.